### PR TITLE
chore(amplify_auth_cognito) update changelog

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Updates auth plugin changelog.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by Amplifiyer** [comment]
👍 Looks good. you need to add it to the interface package as well

File: packages/amplify_auth_cognito/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 0.0.1-dev.4 (2020-12-03)
+
+### Bug Fixes
+
+* Updated amplify-android library version to v1.6.6
+* Updated amplify-ios library version to v1.4.4
+* Fixed bugs preventing guest credentials from being provided from native code to Dart
+* Fixed bugs preventing access tokens from being provided from native code to Dart
+* Fixed bugs preventing user pool tokens from being provided from native code to Dart when configuration includes Cognito User Pools without Cogito Identity Pools.
```
Comment (Amplifiyer) [inline packages/amplify_auth_cognito/CHANGELOG.md:9]: Can we mention the issue Id here?

Comment (haverchuck) [reply packages/amplify_auth_cognito/CHANGELOG.md]: @Amplifiyer Done.

**Review by Amplifiyer — APPROVED** [review]
LGTM

**Pull request merged**